src/screens/proposals/index.tsx

```diff
@@ -36,8 +39,13 @@ const Proposals = () => {
         <div className={classnames('mobile-tablet-header')}>
           <HeaderBarMobile title={t('title')} communityPool={communityPool} />
         </div>
-        <div className={classnames('proposals-content')}>
-          Proposals table
+        <div className={classnames('toggle')}>
+          <div className={classnames('toggleContent')}>
```

Comment:
move the css in `toggleContent` in to `toggle` i don't see a use for `toggleContent`